src/frontend/js/utils/test/mockProduct.ts

```diff
@@ -0,0 +1,13 @@
+import fetchMock from 'fetch-mock';
+import { Order, Product } from 'types/Joanie';
+import { ProductFactory } from 'utils/test/factories/joanie';
+
+export const mockProduct = (order: Order) => {
+  const product: Product = ProductFactory.generate();
+  product.id = order.product;
```

Comment:
With the brand new factory 😎

```suggestion
  const product = ProductFactory({
    id: order.product,
  }).one();
```